### PR TITLE
Add jobs and jupyter documentation

### PR DESCRIPTION
**Pull request by tkakantousis**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [ ] Passes the tests
- [ ] HOPSWORKS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: docs/compute/jobs.md [hunk 1]
```diff
@@ -1 +1,266 @@
...
+    <a  href="../../assets/images/newjob1.png">
+      <img src="../../assets/images/newjob1.png" alt="New job quickstart">
+    </a>
+    <figcaption>New job quickstart</figcaption>
+  </figure>
+</p>
+
+
+### New job from scratch
+In addition to te options available with the new job quickstart, from this menu you can set default arguments to be given as input to the job when
```
Comment (robzor92) [inline docs/compute/jobs.md:42]:
```suggestion
In addition to the options available with the new job quickstart, from this menu you can set default arguments to be given as input to the job when
```

File: docs/compute/jobs.md [hunk 2]
```diff
@@ -1 +1,266 @@
...
+
+### Alerts
+
+You can attach Hopsworks alerts on a per-job basis if you want to receive notifications for job-specific events.
+Firstly, a Hopsworks administrator needs to setup alert channels from the `Cluster settings` page. Currently supposrted channels are
+- email
+- slack
+- pagerduty
+
+Once you have configured the alert channels, you can proceed by specifying alert `receivets` from the project's settings page.
```
Comment (robzor92) [inline docs/compute/jobs.md:158]:
```suggestion
Once you have configured the alert channels, you can proceed by specifying alert `receivers` from the project's settings page.
```

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: docs/compute/jobs.md [hunk 3]
```diff
@@ -1 +1,266 @@
...
+- Logs: Shows the aggregated stdout/stderr logs of the execution. These logs are stored under the `Logs` dataset.
+
+#  Hopsworks IDE Plugin
+
+It is also possible to work on jobs while developing in your IntelliJ/PyCharm IDE by installing the Hopsworks Plugin from the marketplace.
+
+**Usage**
+
+- Open the **Hopsworks Job Preferences** UI for specifying user preferences under **Settings -> Tools -> Hopsworks Job Preferences**.
+- Input the Hopworks project preferences and job details you wish to work on.
```
Comment (robzor92) [inline docs/compute/jobs.md:247]:
```suggestion
- Input the Hopsworks project preferences and job details you wish to work on.
```

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: docs/compute/jobs.md [hunk 4]
```diff
@@ -1 +1,266 @@
...
+All members of a project in Hopsworks can launch the following types of applications through a project's Jobs service:
+
+- Python (*Hopsworks Enterprise only*)
+- Apache Spark
+- Apache Flink
+- Docker (*Hopsworks Enterprise only*)
+
+Launching a job of any type is very similar process, what mostly differs between job types is
+the various configuration parameters each job type comes with. Details on running Python and Docker programs are
+provided in the Python section below.
```
[the remark this answers is not in the document]
Comment (tkakantousis) [reply docs/compute/jobs.md]: fixed

File: docs/compute/jobs.md [hunk 5]
```diff
@@ -1 +1,266 @@
...
+    <a  href="../../assets/images/newjob5.png">
+      <img src="../../assets/images/newjob5.png" alt="Python job configuration properties">
+    </a>
+    <figcaption>Python job configuration properties</figcaption>
+  </figure>
+</p>
+
+
+You do not have to upload the Python program via the Hopsworks UI to run it. That can be done so from within the Python program by using
+``upload`` function of the ``dataset`` module of the ``hops`` Python library http://hops-py.logicalclocks.com
```
[the remark this answers is not in the document]
Comment (tkakantousis) [reply docs/compute/jobs.md]: fixed

File: docs/compute/jobs.md [hunk 6]
```diff
@@ -1 +1,265 @@
...
+**Usage**
+
+- Open the **Hopsworks Job Preferences** UI for specifying user preferences under **Settings -> Tools -> Hopsworks Job Preferences**.
+- Input the Hopsworks project preferences and job details you wish to work on.
+- Open a Project and within the Project Explorer, right click on the program ( .jar, .py, .ipynb) you wish to execute as a job on Hopsworks. Different job actions possible are available in the context menu ( Create, Run, Stop, etc.)
+
+**Actions**
+
+- **Create:** Create or update job as specified in Hopsworks Job Preferences
+- **Run:** Uploads the program first to the HDFS path as specficied and runs job
```
Comment (robzor92) [inline docs/compute/jobs.md:252]:
```suggestion
- **Run:** Uploads the program first to the HDFS path as specified and runs job
```

File: docs/compute/jupyter.md [hunk 7]
```diff
@@ -1 +1,272 @@
...
+    </a>
+    <figcaption>Jupyter dashboard page showing currently running applications</figcaption>
+  </figure>
+</p>
+
+## Recent notebooks
+
+When you run a notebook, its Jupyter configuration is stored and attached to the notebook.
+You can use this configuration later to start the Jupyter notebook directly from the Jupyter dashboard by clicking 
+`Open in Juptyer` under the `Recent notebooks` section of the dashboard. For example, in the image above you can see a
```
Comment (robzor92) [inline docs/compute/jupyter.md:59]:
```suggestion
`Open in Jupyter` under the `Recent notebooks` section of the dashboard. For example, in the image above you can see a
```

File: docs/compute/jupyter.md [hunk 8]
```diff
@@ -1 +1,272 @@
-# Jupyter docs
\ No newline at end of file
+# Jupyter notebooks
+
+Developing with Jupyter notebooks is provided as a service in Hopsworks as part the compute section of a project which you
```
Comment (robzor92) [inline docs/compute/jupyter.md:3]:
```suggestion
Developing with Jupyter notebooks is provided as a service in Hopsworks as part of the compute section of a project which you 
```

File: docs/compute/jupyter.md [hunk 9]
```diff
@@ -1 +1,272 @@
...
+    </a>
+    <figcaption>Classic Jupyter</figcaption>
+  </figure>
+</p>
+
+
+Jupyter server is using the same configuration options as the Jobs service to run notebooks. Currently, the `Spark/PySpark` 
+and `Python` configuration options are supported. For more information on how to set these options, see the [Jobs](./jobs.md) section 
+and for more information on how to work with Spark and Jupyter in Hopsworks see the `Jupyter + Spark on Hopsworks` below. 
+If you would like to switch between configurations, you will to shut down Jupyter server first.
```
Comment (robzor92) [inline docs/compute/jupyter.md:37]:
```suggestion
If you would like to switch between configurations, you will have to shut down the running Jupyter server first.
```

File: docs/compute/jupyter.md [hunk 10]
```diff
@@ -1 +1,272 @@
...
+<p align="center">
+  <figure>
+    <a  href="../../assets/images/jupyter9.png">
+      <img src="../../assets/images/jupyter9.png" alt="Printing a list of all sparkmagic commands">
+    </a>
+    <figcaption>Printing a list of all sparkmagic commands</figcaption>
+  </figure>
+</p>
+
+### Plotting with Pyspark Kernel
```
Comment (robzor92) [inline docs/compute/jupyter.md:188]:
```suggestion
### Plotting with PySpark Kernel
```

**Review by robzor92 — CHANGES_REQUESTED** [review]

**Review by robzor92 — APPROVED** [review]

**Pull request merged**